drivers/client_access_control/app/views/client_access_control/clients/enrollment_details/_enrollment_rollups.haml

```diff
@@ -27,3 +27,6 @@
     %article.rollup.o-dashboard__block.o-dashboard__block--secondary{data: {partial: 'hmis_clients'}}
       %header
         %h1.h3 HMIS Clients (from ETO)
+    %article.rollup.o-dashboard__block.o-dashboard__block--secondary{data: {partial: 'disabilities'}}
+      %header
+        %h1.h3 Disabilities
```

Comment:
Let's move this up between Health and DV and Enrollment CoCs.

Reply:
✅